garak/generators/litellm.py

```diff
@@ -96,7 +92,6 @@ class LiteLLMGenerator(Generator):
         "generations",
         "context_len",
         "max_tokens",
-        "api_key",
```

Comment:
This should still be a `supported_param`, when `_supported_params` is defined any parameter not listed is filtered from configuration files.

Reply:
Added api key back as a supported param @jmartin-tech